Source/WebCore/accessibility/AXObjectCache.h

```diff
@@ -256,6 +256,12 @@ class AccessibilityReplacedText {
     WEBCORE_AXNOTIFICATION_KEYS_DEFAULT(macro)
 #endif
 
+enum class AXNotification {
+#define WEBCORE_DEFINE_AXNOTIFICATION_ENUM(name) AX##name,
+WEBCORE_AXNOTIFICATION_KEYS(WEBCORE_DEFINE_AXNOTIFICATION_ENUM)
```

Comment:
Now that this is an enum class, the `AX` notification on all the keys is probably redundant, e.g. I think they should be:

`AXNotification::VisibilityChanged` vs. `AXNotification::AXVisibilityChanged`

But I can take care of that in a follow-up.

Reply:
https://github.com/WebKit/WebKit/pull/37229